### DISCUSSION TITLE
[Question] My ground terrain get so dark

### DISCUSSION THREAD

**Discussion by KUsaka-xu** [isaac-sim/IsaacLab#725] [Q&A]
At first it was OK, but I did not why it became so dark, can someone kindly help me?
![Screenshot from 2024-07-24 14-54-28](https://github.com/user-attachments/assets/91bcc335-bb27-4335-b744-306da7fe4d89)
![Screenshot from 2024-07-23 19-37-41](https://github.com/user-attachments/assets/bca9d0d1-22c1-4e13-ac83-0af3a4e69d3b)

**Comment by Mayankm96** [comment]
Very hard to comment on this without a proper repro or understanding of your setup.
